GEMINI.md

```
@@ -0,0 +1,23 @@
+# 回答言語およびプロジェクトドキュメント保存ルール
+
+## 基本方針
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)